examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -124,6 +126,17 @@ CHIP_ERROR CommonAttributeAccessInterface::Write(const chip::app::ConcreteDataAt
     return c->ForwardWriteToBridge(aPath, aDecoder);
 }
 
+CHIP_ERROR CommonAttributeAccessInterface::Read(const chip::app::ConcreteReadAttributePath & aPath, chip::TLV::TLVWriter & writer)
+{
+    CommonCluster * c = FindCluster(aPath);
+    if (!c)
+        return CHIP_ERROR_NOT_IMPLEMENTED;
```

Comment:
is not implemented the right call here and below? Or would some `CHIP_ERROR_NOT_FOUND`  or `CHIP_ERROR_KEY_NOT_FOUND` be more appropriate?

Reply:
Addressed in [37ff981](https://github.com/project-chip/connectedhomeip/pull/23840/commits/37ff98112ffb2b2d0dd795ad089f2311d6a417ab)